test-requirements.txt

```diff
@@ -4,5 +4,6 @@ flake8; python_version >= '2.7'
 mock==2.0.0; python_version == '2.6'
 mock==3.0.5; python_version >= '2.7' and python_version <= '3.5'
 mock==4.0.2; python_version >= '3.6'
+distro; python_version >= '3.7'
```

Comment:
>distro is supported and tested on Python 2.7, 3.4+ 

Got this from their page. We should modify the line to download distro for python_version >= 2.7

Reply:
i'm thinking we should match this to the images tested in DCR to make UTs closer to actual VM environments. We know Ubuntu 20 will be 3.8 and includes 'distro' by default... other distros?

Reply:
Since our `requirements.txt` file already specify `distro` is a dependency, this requirements in `test-requirements.txt` is in sync.

Note that I've also updated the requirement in `requirements.txt` to start with py3.7 (as opposed to 3.8) since `platform.dist` is deprecated starting 3.7.

Reply:
Ignore this, it's deprecated starting 3.8, not 3.7, reverting that back.